L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -446,46 +449,59 @@ void PrimitiveConversion::convert_rpc(
   int tp_station   = tp_detId.station();    // 1 - 4
   int tp_ring      = tp_detId.ring();       // 2 - 3 (increasing theta)
   int tp_roll      = tp_detId.roll();       // 1 - 3 (decreasing theta; aka A - C; space between rolls is 9 - 15 in theta_fp)
-  // int tp_layer     = tp_detId.layer();
+  //int tp_layer     = tp_detId.layer();
 
   int tp_bx        = tp_data.bx;
   int tp_strip     = ((tp_data.strip_low + tp_data.strip_hi) / 2);  // in full-strip unit
   int tp_valid     = tp_data.valid;
 
   const bool is_neighbor = (pc_station == 5);
 
+  // CSC-like sector, subsector and chamber numbers
+  int csc_tp_chamber   = (tp_sector - 1)*6 + tp_subsector;
+  int csc_tp_sector    = (tp_subsector > 2) ? tp_sector : ((tp_sector + 4) % 6) + 1;  // Rotate by 20 deg
+  int csc_tp_subsector = ((tp_subsector + 3) % 6) + 1;  // Rotate by 2
+
   // Set properties
   conv_hit.SetRPCDetId       ( tp_detId );
 
   conv_hit.set_endcap        ( (tp_endcap == 2) ? -1 : tp_endcap );
   conv_hit.set_station       ( tp_station );
   conv_hit.set_ring          ( tp_ring );
   conv_hit.set_roll          ( tp_roll );
+  conv_hit.set_chamber       ( csc_tp_chamber );
+  conv_hit.set_sector        ( csc_tp_sector );
+  conv_hit.set_subsector     ( csc_tp_subsector );
+  //conv_hit.set_csc_ID        ( tp_csc_ID );
```

Comment:
Remove commented-out lines.

Reply:
I keep these lines to list all the possible setters provided by the EMTFHit class. This makes it easier to remember to set it every time a new setter is added to the class. As you can just CTRL+F one of the existing setters, and just duplicate the line. I think it's good practice.

Reply:
Fair enough.